src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    private String taskName;
+    private boolean isDone;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false; // Default to not done
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
```

Comment:
Output of markTaskAsDone() in main can be placed in this function. Similarly, this can be done for unmarkAsDone().